ts/main.ts

```diff
@@ -19,33 +17,12 @@ export async function main(options?: {
     //     return ThreadID.fromString(threadIDString)
     // }
 
-    // let database: Database
-    // if (options?.local) {
-    //     const identity = await Database.randomIdentity()
```

Comment:
We learned that the Database class wasn't the right method and was proving confusing to users. We are moving all the local methods directly _into_ the Client as part of this, https://github.com/textileio/js-threads/issues/414. So for now, just use Client and then it should pretty cleanly upgrade to just being local at the same time once that ticket is wrapped.